Updated test.go to match recent changes

ccicchitelli opened the pull request:
The test.go file currently will not run, this update fixes that.

ccicchitelli committed:
Remove LDFLAGS as they weren't being imported, TEST ONLY

ccicchitelli committed:
Revert previous checkin

ccicchitelli committed:
Updated test file with @bonanzaeric's update

ccicchitelli committed:
Updated mpu.InitMPU call to match latest changes

jpoirier commented:
fyi - if you remove those flags mpu.go won't build. mpu.go is a Go language wrapper around a library file that was built using C. Go and C have two different ABIs and Go uses a tool called cgo for its cffi (foreign function interface), cgo needs the ld flags so it knows what lib to look for.

ccicchitelli commented:
Yeah I figured that out...i had removed them to try to generate a new error, which eventually it did.

jpoirier commented:
yup, saw the updated commit but thought I'd explain why the magical commands are there.

ccicchitelli commented:
thx!!

Commit:
Merge remote-tracking branch 'refs/remotes/cyoung/master'

ccicchitelli committed:
increased allowed update rate in mpu9150 code

ccicchitelli committed:
test changing makefile for mpu to 9250

ccicchitelli committed:
End for loop in test after 10 tries

ccicchitelli committed:
changed Makefile-native-shared to:

DEFS = -DEMPL_TARGET_LINUX -DMPU9150 -DAK8975_SECONDARY
#DEFS = -DEMPL_TARGET_LINUX -DMPU9250 -DAK8963_SECONDARY

ccicchitelli committed:
added support for this chip revision of MPU6050

ccicchitelli committed:
testing fix for dmp_load_motion_driver_firmware error

ccicchitelli committed:
bypassing error from dmp_set_fifo_rate, as we're not using DMP

ccicchitelli committed:
test fix

ccicchitelli committed:
updated test.go

ccicchitelli committed:
test

ccicchitelli committed:
changed sample rate to 500

ccicchitelli committed:
test fix

ccicchitelli committed:
test fix

ccicchitelli committed:
revert previous

ccicchitelli committed:
test bypass of data_ready()
